informer/informer_integration_test.go

```diff
@@ -20,30 +20,30 @@ package informer
 */
```

Comment:
The test watches for ConfigMaps in a namespace. So it don't have to ensure CRD and cleanup is simply namespace deletion.